### PR TITLE
fix deployment_tools in docs

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
remove old and deprecated deployment_tools path

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
also <omz-repo>\ci\update-requirements.py should be changed

File: tools/accuracy_checker/configs/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ See how to evaluate model with using predefined configuration file for [densenet
     ```
 3. Convert model in the Inference Engine IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)
     ```sh
-    OMZ_ROOT/tools/model_tools/converter.py --name densenet-121-tf --download_dir MODEL_DIR --mo OPENVINO_DIR/deployment_tools/model_optimizer/mo.py
+    OMZ_ROOT/tools/model_tools/converter.py --name densenet-121-tf --download_dir MODEL_DIR
```
Comment (eaidova) [inline tools/accuracy_checker/configs/README.md:83]:
```suggestion
    omz_converter --name densenet-121-tf --download_dir MODEL_DIR
```

File: tools/accuracy_checker/configs/README.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ See how to evaluate model with using predefined configuration file for [densenet
     ```
 3. Convert model in the Inference Engine IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)
```
Comment (eaidova) [inline tools/accuracy_checker/configs/README.md:81]:
```suggestion
3. Convert model in the OpenVINO IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)
```

File: tools/accuracy_checker/configs/README.md (hunk 1 above)
Comment (vladimir-dudnik) [reply tools/accuracy_checker/configs/README.md]: accepted

File: tools/accuracy_checker/configs/README.md (hunk 2 above)
Comment (vladimir-dudnik) [reply tools/accuracy_checker/configs/README.md]: accepted

**Review by eaidova — APPROVED** [review]

**Review by jkamelin — APPROVED** [review]

File: tools/accuracy_checker/configs/README.md [hunk 3]
```diff
@@ -78,9 +78,9 @@ See how to evaluate model with using predefined configuration file for [densenet
     ```sh
     OMZ_ROOT/tools/model_tools/downloader.py --name densenet-121-tf --output_dir MODEL_DIR
```
Comment (eaidova) [inline tools/accuracy_checker/configs/README.md:79]:
```suggestion
    omz_downloader --name densenet-121-tf --output_dir MODEL_DIR
```

File: ci/requirements-conversion.txt [hunk 4]
```diff
@@ -22,10 +22,10 @@ decorator==4.4.2
...
 defusedxml==0.7.1
     # via
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_caffe.txt
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_mxnet.txt
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_onnx.txt
-    #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_tf2.txt
+    #   -r <openvino repository>/tools/mo/requirements_caffe.txt
+    #   -r <openvino repository>/tools/mo/requirements_mxnet.txt
+    #   -r <openvino repository>/tools/mo/requirements_onnx.txt
+    #   -r <openvino repository>/tools/mo/requirements_tf2.txt
```
Comment (Wovchena) [inline ci/requirements-conversion.txt:28]: The file is generated automatically by https://github.com/openvinotoolkit/open_model_zoo/blob/master/ci/update-requirements.py. You need to update the script

Comment (ilya-lavrenov) [reply ci/requirements-conversion.txt:28]: these locations are nor valid anyway. MO is not part of OpenVINO Installer, it's installed only via PyPi with dependencies specified using extra notation

**Review by ilya-lavrenov — CHANGES_REQUESTED** [review]

File: ci/requirements-conversion.txt (hunk 4 above)
Comment (Wovchena) [reply ci/requirements-conversion.txt:28]: These ci/requirements* are only used in our CI. They aren't redistributed. If you want to update them, you should target 21.2. As far as I remember, 21.2 minimal python version is going to be 3.8. If ov has already moved to python3.8, you can update the requirements, if not, we shouldn't bother for now.

File: demos/README.md [hunk 5]
```diff
@@ -253,7 +253,7 @@ a solution for a demo code. Optionally, you can also specify the preferred Micro
 versions are: `VS2017`, `VS2019`. For example, to build the demos using the Microsoft Visual Studio 2017, use the following command:
 
 ```bat
-build_demos_msvc.bat VS2017
+build_demos_msvc.bat VS2019
```
[the remark this answers is not in the document]
Comment (vladimir-dudnik) [reply demos/README.md:254]: yes, removed

File: tools/accuracy_checker/configs/README.md [hunk 6]
```diff
@@ -78,9 +78,9 @@ See how to evaluate model with using predefined configuration file for [densenet
...
     ```
-3. Convert model in the Inference Engine IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)
+3. Convert model in the OpenVINO IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)
     ```sh
-    OMZ_ROOT/tools/model_tools/converter.py --name densenet-121-tf --download_dir MODEL_DIR --mo OPENVINO_DIR/deployment_tools/model_optimizer/mo.py
+    omz_converter --name densenet-121-tf --download_dir MODEL_DIR
     ```
 4. Run evaluation for model in FP32 precision using [Accuracy Checker](../README.md)
     ```sh
```
Comment (slyubimt) [inline tools/accuracy_checker/configs/README.md:86]: Should we use omz_downloader and  omz_quantizer aliases instead of scripts everywhere on this page?

Comment (vladimir-dudnik) [reply tools/accuracy_checker/configs/README.md:86]: yes, we should

**Review by Wovchena — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
merge with known non relevant CI issues with media codec

**Pull request merged**